pyomo/contrib/mindtpy/single_tree.py

```diff
@@ -657,256 +651,143 @@ def handle_lazy_subproblem_other_termination(
                 'condition of {}'.format(termination_condition)
             )
 
-    def handle_lazy_regularization_problem(
```

Comment:
We don't need this anymore?

Reply:
Yes. Now we use the method inside algorithm base class instead.